<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstDialog</class>
 <widget class="QDialog" name="FirstDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT CK2 Launcher configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowsePathSteam">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPathSteam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelSteam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Steam location</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPathGame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowsePathGame">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Game location</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>170</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowsePathConfig">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPathConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelConfig">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Config location</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxNoSteam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Disable steam-related functions</string>
   </property>
   <property name="text">
    <string>No steam mode</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAdvancedMode">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include experimental functions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Enable advanced mode</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FirstDialog</receiver>
   <slot>buttonClickedOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FirstDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxPathSteam</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FirstDialog</receiver>
   <slot>indexChangedSteam(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowsePathSteam</sender>
   <signal>clicked()</signal>
   <receiver>FirstDialog</receiver>
   <slot>buttonClickedBrowseSteam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowsePathGame</sender>
   <signal>clicked()</signal>
   <receiver>FirstDialog</receiver>
   <slot>buttonClickedBrowseGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowsePathConfig</sender>
   <signal>clicked()</signal>
   <receiver>FirstDialog</receiver>
   <slot>buttonClickedBrowseConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxNoSteam</sender>
   <signal>toggled(bool)</signal>
   <receiver>FirstDialog</receiver>
   <slot>checkedNoSteam(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>indexChangedSteam(int)</slot>
  <slot>buttonClickedBrowseSteam()</slot>
  <slot>buttonClickedBrowseGame()</slot>
  <slot>buttonClickedBrowseConfig()</slot>
  <slot>checkedNoSteam(bool)</slot>
  <slot>buttonClickedOk()</slot>
 </slots>
</ui>
